mongodbatlas/table_mongodbatlas_cluster.go

```diff
@@ -0,0 +1,272 @@
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+func tableMongoDBAtlasCluster(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "mongodbatlas_cluster",
+		Description: "MongoDB Atlas Cluster is a NoSQL Database-as-a-Service offering in the public cloud (available in Microsoft Azure, Google Cloud Platform, Amazon Web Services).",
+		List: &plugin.ListConfig{
+			Hydrate: listAtlasClusters,
+		},
+		Get: &plugin.GetConfig{
+			Hydrate:    getAtlasCluster,
+			KeyColumns: plugin.AllColumns([]string{"name"}),
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "id",
+				Description: "Unique 24-hexadecimal digit string that identifies the cluster.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "name",
+				Description: "The name of the cluster as it appears in Atlas.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "project_id",
+				Description: "Unique identifier of the project that this cluster belongs to",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("GroupID"),
+			},
+			{
+				Name:        "auto_scaling",
+				Description: "TDB",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "bi_connector_config",
+				Description: "Configuration settings applied to BI Connector for Atlas on this cluster.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("BiConnector"),
+			},
+			{
+				Name:        "cluster_type",
+				Description: "TDB",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "disk_size_gb",
+				Description: "Capacity, in gigabytes, of the host's root volume. Increase this number to add capacity, up to a maximum possible value of 4096 (i.e., 4 TB). This value must be a positive number.",
+				Type:        proto.ColumnType_DOUBLE,
+				Transform:   transform.FromField("DiskSizeGB"),
+			},
+			{
+				Name:        "encryption_at_rest_provider",
+				Description: "Cloud service provider that offers Encryption at Rest.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "labels",
+				Description: "Collection of key-value pairs that tag and categorize the cluster. Each key and value has a maximum length of 255 characters.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "mongo_db_version",
+				Description: "Version of MongoDB that the cluster is running, in X.Y.Z format.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("MongoDBVersion"),
+			},
+			{
+				Name:        "mongodb_major_version",
+				Description: "MongoDB Version of the cluster.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("MongoDBMajorVersion"),
+			},
+			{
+				Name:        "mongo_uri",
+				Description: "TDB",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("MongoURI"),
```

Comment:
Do we still need this transform here since we're now default to `transform.FromGo()`?